lib/testing/src/lib/content-services/dialog/content-node-selector-dialog.page.ts

```diff
@@ -99,13 +101,32 @@ export class ContentNodeSelectorDialogPage {
     }
 
     async clickContentNodeSelectorResult(name): Promise<void> {
-        await this.contentList.dataTablePage().clickRowByContent(name);
+        await this.dataTable.clickRowByContent(name);
     }
 
     async doubleClickContentNodeSelectorResult(name): Promise<void> {
         // First click to select from search mode and second click to actually open node
-        await this.contentList.dataTablePage().doubleClickRowByContent(name);
-        await this.contentList.dataTablePage().doubleClickRowByContent(name);
+        await this.dataTable.doubleClickRowByContent(name);
```

Comment:
you're double-clicking twice here ?

Reply:
this code was there already and I only changed this.contentList.dataTablePage() => this.dataTable. Btn it has a valid comment ( see line 108)

Reply:
I can see that the code was already there, but do you think it's correct to doubleClick twice?

Reply:
@adinapitul can you raise a different bug with more context? In this way, we can discuss if the new behaviour is correct